index.html

```diff
@@ -13,7 +16,21 @@
     <meta name="apple-touch-fullscreen" content="yes" />
     <meta name="apple-mobile-web-app-title" content="Terminal7" />
     <meta name="apple-mobile-web-app-capable" content="yes" />
+
     <script type="module" src="/main.ts"></script>
+
+    <script>
```

Comment:
I don't believe we need this script anymore.

Reply:
Done

Reply:
It's still here